README.md

```diff
@@ -74,6 +74,15 @@ Godot 4.0 (master branch) @ commit https://github.com/godotengine/godot/commit/d
 - Support for building on 3.x has been dropped and no new features are being pushed
 	- Godot RE Tools still retains the ability to decompile 3.x and 2.x projects, however.
 
+### Standalone
+
+Assuming you compiled with `scons platform=linuxbsd target=template_debug`,
+
+```bash
+cd modules/gdsdecomp/standalone
+$ ../../../bin/godot.linuxbsd.template_debug.x86_64.llvm --headless --recover=...
```

Comment:
```suggestion
bin/godot.linuxbsd.template_debug.x86_64.llvm --headless --path modules/gdsdecomp/standalone --recover=<pck/apk/exe>
```
This makes it easier to just launch the thing as a single line in VS Code or other IDEs.

Reply:
done!